docs/spelling_wordlist.txt

```diff
@@ -381,6 +381,7 @@ dagruns
 DagRunState
 dagRunState
 DAGs
```

Comment:
Thinking out loud: I wonder if removing `DAGs` and `DAG` from the docs spellcheck will help find any other instances that were missed

Reply:
this is left alone because I am not changing things like existing release notes.